ownCloud/Client/Actions/Actions+Extensions/MediaEditingAction.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  MediaEditingAction.swift
+//  ownCloud
+//
+//  Created by Matthias Hühne on 23/01/2020.
+//  Copyright © 2020 ownCloud GmbH. All rights reserved.
+//
+
+/*
+* Copyright (C) 2020, ownCloud GmbH.
+*
+* This code is covered by the GNU Public License Version 3.
+*
+* For distribution utilizing Apple mechanisms please see https://owncloud.org/contribute/iOS-license-exception/
+* You should have received a copy of this license along with this program. If not, see <http://www.gnu.org/licenses/gpl-3.0.en.html>.
+*
+*/
+
+import ownCloudSDK
+
+@available(iOS 13.0, *)
+class MediaEditingAction : MarkupAction {
```

Comment:
@hosy Seems strange to me when `MediaEditingAction` is derived from `MarkupAction`. May be the base class should be something like DocumentEditingAction which btw. could have supportedMimeTypes property and could also have generic implementation of `applicablePosition(forContext:)` which is duplicated here in both actions